pysilcam/silcamgui/interactive_summary.py

```diff
@@ -2,20 +2,23 @@
 import pysilcam.postprocess as scpp
 import pysilcam.oilgas as scog
 from pysilcam.config import PySilcamSettings
+from pysilcam.background import correct_im_fast
+from pysilcam.fakepymba import silcam_load
 from tqdm import tqdm
 import numpy as np
 import cmocean
 import matplotlib.pyplot as plt
 import matplotlib
-from PyQt5.QtWidgets import QMainWindow, QApplication, QAction, QInputDialog, QMessageBox, QFileDialog
+from PyQt5.QtWidgets import *
 from PyQt5 import QtCore
 from PyQt5 import QtWidgets
+from PyQt5.QtGui import *
```

Comment:
I suggest we stick with explicit imports, it makes intension clearer and improved readability.